Client/Frontend/Browser/Playlist/VideoPlayer/VideoPlayerInfoBar.swift

```diff
@@ -0,0 +1,119 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+import AVKit
+
+class VideoPlayerInfoBar: UIView {
+    private let controlStackView = UIStackView().then {
+        $0.spacing = 32.0
+    }
+    
+    private let leftStackView = UIStackView().then {
+        $0.spacing = 32.0
+        $0.alignment = .center
+    }
+    
+    private let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .dark)).then {
```

Comment:
Since we're iOS 13+ now you m ay want to switch to using one of the `systemMaterialDark` variants to get a better blur: https://developer.apple.com/documentation/uikit/uiblureffect/style